src/main/java/org/gelecekbilimde/scienceplatform/post/model/request/CategoryCreateRequest.java

```diff
@@ -9,10 +12,27 @@
 @Setter
 @Builder
 public class CategoryCreateRequest {
-	@NotBlank
+
+	@NotBlank(message = "Category name cannot be blank")
+	@Size(max = 36, message = "Category name must be less than 36 characters")
 	private String name;
+
+	@NotBlank(message = "Description cannot be blank")
+	@Size(max = 255, message = "Description must be less than 255 characters")
+	private String description;
+
+	@NotNull(message = "Order number cannot be null")
+	@PositiveOrZero(message = "Order number must be zero or a positive integer")
```

Comment:
Özel bir mesaj yazmamaya çalışalımi buradaki mesajları bu şekilde özelleştirdiğimizde yazması zor olur ve uyuşmaz. Mümkün olduğunca spring'in kendi varsayılan mesajlarını kullanalım. 

Ek olarak kategori sıra numarası 0 olabilir mi? Yani 0'dan başlayarak mı sıralanacak? Bence 1'den başlamalı ama bunun üzerine ortak bir karara varmamız faydalı olabilir
```suggestion
	@NotNull
	@PositiveOrZero
```